### ISSUE TITLE
Only leveldb_open() is exported

### ISSUE THREAD

**Issue by kprimice** [richfitz/rleveldb#2]
There does not seem to be any function or method exported besides `leveldb_open`

**Comment by richfitz** [comment]
That's right - this package is only part formed - see the tests for example usage (assuming everything is exported).  Unfortunately, with no windows support I could not justify much more development time on this package, as it would be difficult for it to move onto CRAN.

I implemented a similar package as [`thor`](https://github.com/richfitz/thor), which might be of interest
